audio-format-service/tests/test_main.py

```diff
@@ -0,0 +1,297 @@
+"""Integration tests for Audio Format Service
+
+This module contains integration tests for the FastAPI audio format service,
+testing all endpoints and their functionality.
+"""
+
+import io
+import os
+
+import pytest
+from fastapi.testclient import TestClient
+
+# Import the FastAPI app
+import sys
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', 'src'))
```

Comment:
please avoid usage of this pattern with "sys.path.insert" -- and use PYTHONPATH or other way to provide context for modules